src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileServiceImpl.java

```diff
@@ -64,6 +91,24 @@ public ConcurRequestExtractFileValidationService getConcurRequestExtractFileVali
         return this.concurRequestExtractFileValidationService;
     }
 
+    private String parseRequestExtractFileNameFrom(String requestExtractFullyQualifiedFileName) {
+        //plus 1 because we do not want the forward slash
+        return requestExtractFullyQualifiedFileName.substring((requestExtractFullyQualifiedFileName.lastIndexOf(ConcurConstants.FORWARD_SLASH)) + 1);
```

Comment:
If you are questioning the addition of 1, the reason is:
We want just the file name to add to the database table entry to track duplicates.
Method lastIndexOf returns the index of the slash.
Method substring starts at the index provided and goes to the end of the string sent to it.
Thus, the returned value from the substring call was including the forward slash which is not part of the filename.